src/tests/tests.rs

```diff
@@ -0,0 +1,14 @@
+use super::Histogram;
+
+#[path = "helpers.rs"]
```

Comment:
One example when this is really handy is when a bug is found and we need to `bisect` some day far into the future. `blame` also works a lot better when the merge that introduces the change is directly tied to the reason for the change.